### DISCUSSION TITLE
关于新增form表单组件

### DISCUSSION THREAD

**Discussion by eJayYoung** [airyland/vux#3751] [Ideas]
hi，vux组件在移动端我用了两年多了，写表单的需求比较多，所以自己封装了一个[vux-form](https://github.com/eJayYoung/vux-form)表单的组件，希望能够合并到vux中

## 为什么想要合并到vux中呢？
目前我的form组件内置了一些vux的组件，这样导致我的form组件引用了vux，业务中会打包另一份vux，代码体积太大，有点冗余

## 目前已经实现了哪些功能？
- 表单的统一校验和提示交互
- 表单支持内置组件和自定义组件两种模式
- 内置组件支持命名slot传递

best regard！盼回复！

**Comment by caozhong1996** [comment]
非常的欢迎你来提交pr，我会帮你review代码。
